pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -30,6 +30,21 @@ import (
 // Code is the Status code/type which is returned from plugins.
 type Code int
 
+// NodeScore represents the score of a particular node. The higher the score, the higher
+// the node is ranked during scheduling.
+type NodeScore struct {
+	// Name of the host
+	Host string
```

Comment:
Please use `Node` instead of `Host`. `Host` is not a well-defined term in K8s. I know that Host is used in some parts of the code-base, but let's not add any more.